build/windows/upload_msi.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+#
+#
+# Upload msi artifacts to GH Release assets
+#
+#
+INTEGRATION=$1
+ARCH=$2
+TAG=$3
+
+hub release edit -a "build/package/windows/nri-${ARCH}-installer/bin/Release/nri-${INTEGRATION}-${ARCH}.${TAG:1}.msi" -m ${TAG} ${TAG}
```

Comment:
why do we use `hub` here and not `gh` like we do for linux? is `hub` installed by default in the windows runners?

Reply:
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md `hub` is there, but looks like we just forgot to migrate to the same tool. 
This 2 already in win2019 box by default
- Hub CLI 2.14.2
- GitHub CLI 1.3.1

Reply:
I'd pick whichever is being used in more places (`gh` if I'm not mistaken?) and avoid re-installing it, just to keep consistency across all pipelines.

Reply:
https://github.com/cli/cli/blob/trunk/docs/gh-vs-hub.md

hub has been basically abandoned by github, `gh` is maintained by github people